client/src/components/Confirmation.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { keyframes } from "styled-components";
+
+export const flyFromRight = keyframes`
```

Comment:
`Confirmation` is a copy of `Alert`. I would suggest to add a property to change the color.